pom.xml

```diff
@@ -625,6 +625,12 @@
                                         <exclude>META-INF/services/com.fasterxml.jackson.*</exclude>
                                     </excludes>
                                 </filter>
+                                <filter>
+                                    <artifact>com.microsoft.azure:azure-data-lake-store-sdk</artifact>
+                                    <excludes>
+                                        <exclude>com/microsoft/azure/datalake/store/HttpTransport.class</exclude>
```

Comment:
> Extracted from https://raw.githubusercontent.com/Azure/azure-data-lake-store-java/2.2.9/src/main/java/com/microsoft/azure/datalake/store/HttpTransport.java

Can we have some guard which will tell us when we need to bump to newer version (if we bump  dep.hadoop.version)

Reply:
I don't think so. We don't have guards for the rest of classes.